.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 99
+  groups:
+    github-actions:
+      patterns:
+        - "*" # All possible actions in single PR
```

Comment:
Probably, grouping is not a good idea. Seems like Centos7 cannot handle the latest  `checkout` action properly.